lerobot/configs/env/aloha.yaml

```diff
@@ -2,6 +2,12 @@
 
 fps: 50
 
+eval:
+  # `use_async_envs` specifies whether to use asynchronous environments (multiprocessing).
+  # set it to false to avoid EGL problem of the ACT env
```

Comment:
I'd rather not say this is an EGL problem without verifying that is truly the case. I'm not 100% sure it is. Maybe @aliberts knows?

Reply:
I just tried running it and this is the error I see:

```
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":1"
      after 135 requests (135 known processed) with 0 events remaining.
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
X connection to :1 broken (explicit kill or server shutdown).
Stepping through eval batches:   0%|                                                     | 0/1 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/home/alexander/Projects/lerobot/lerobot/scripts/eval.py", line 572, in <module>
    main(
  File "/home/alexander/Projects/lerobot/lerobot/scripts/eval.py", line 483, in main
    info = eval_policy(
  File "/home/alexander/Projects/lerobot/lerobot/scripts/eval.py", line 282, in eval_policy
    rollout_data = rollout(
  File "/home/alexander/Projects/lerobot/lerobot/scripts/eval.py", line 122, in rollout
    observation, info = env.reset(seed=seeds)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/vector_env.py", line 140, in reset
    return self.reset_wait(seed=seed, options=options)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py", line 249, in reset_wait
    results, successes = zip(*[pipe.recv() for pipe in self.parent_pipes])
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py", line 249, in <listcomp>
    results, successes = zip(*[pipe.recv() for pipe in self.parent_pipes])
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/multiprocessing/connection.py", line 250, in recv
    buf = self._recv_bytes()
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/multiprocessing/connection.py", line 414, in _recv_bytes
    buf = self._recv(4)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/multiprocessing/connection.py", line 383, in _recv
    raise EOFError
EOFError
/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py:460: UserWarning: WARN: Calling `close` while waiting for a pending call to `reset` to complete.
  logger.warn(
Exception ignored in: <function AsyncVectorEnv.__del__ at 0x75b371771a20>
Traceback (most recent call last):
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py", line 549, in __del__
    self.close(terminate=True)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/vector_env.py", line 272, in close
    self.close_extras(**kwargs)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py", line 464, in close_extras
    function(timeout)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py", line 249, in reset_wait
    results, successes = zip(*[pipe.recv() for pipe in self.parent_pipes])
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/site-packages/gymnasium/vector/async_vector_env.py", line 249, in <listcomp>
    results, successes = zip(*[pipe.recv() for pipe in self.parent_pipes])
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/multiprocessing/connection.py", line 250, in recv
    buf = self._recv_bytes()
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/multiprocessing/connection.py", line 414, in _recv_bytes
    buf = self._recv(4)
  File "/home/alexander/anaconda3/envs/lerobot/lib/python3.10/multiprocessing/connection.py", line 383, in _recv
    raise EOFError
EOFError:
```

maybe not EGL.

Reply:
I see and let me modify it